slu-scripts/auth-proof/auth-proof.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs';
```

Comment:
Please remove your catches from the functions this should be handled by the calling tool or script